application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadResetActivityCommand.java

```diff
@@ -0,0 +1,54 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.entities.ISnowflake;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.requests.restaction.pagination.MessagePaginationAction;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * This command will reset the activity in a help thread.
+ *
+ * @author mDyingStar
+ */
+public final class HelpThreadResetActivityCommand extends SlashCommandAdapter {
```

Comment:
sorry we forgot to update the issue but u'll need to create this as a subcommand of already existing help-thread command

Reply:
I thought so. I would have clarified that, which is why the pr is in draft mode.